docs/reference/ccr/auto-follow.asciidoc

```diff
@@ -7,6 +7,12 @@ each new index in the series is replicated automatically. Whenever the name of
 a new index on the remote cluster matches the auto-follow pattern, a
 corresponding follower index is added to the local cluster.
 
+NOTE: auto-follow patterns creates follower indices only for opened, fully
+assigned indices that match the specified pattern in the remote cluster. The
+ patterns should not match special indices like <<frozen-indices,frozen indices>>,
+<<searchable-snapshots,{search-snaps}>> or <<index-blocks-read-only, read-only indices>>
+that cannot be used for CCR purpose.
```

Comment:
I'd streamline the wording a little:
```suggestion
NOTE: Auto-follow patterns only match regular open indices on the remote
cluster. Auto-follow patterns do not match indices that can't be used for
{ccr-init}, such as <<frozen-indices,frozen indices>>,
<<searchable-snapshots,{search-snaps}>>, or <<index-blocks-read-only,read-only
indices>>.
```

In the original copy, I wasn't sure what `fully assigned` meant. I _think_ this means the index's data is completely stored on local shards (as opposed to searchable snapshots using a shared cache with a repo). If that's the case, I think we can use `regular` index. If not, let me know.

Reply:
"fully assigned" is misleading, sorry, I should have written "with all primary shards started".

Reply:
Thanks for the answer. I amended my suggestion.